background/background.go

```diff
@@ -279,10 +276,10 @@ func (n *NotificationHandler) Close() error {
 	return nil
 }
 
-func connectNotifier(dcrdWithNotifs rpc.DcrdConnect) error {
+func connectNotifier(shutdownCtx context.Context, dcrdWithNotifs rpc.DcrdConnect) error {
 	notifierClosed = make(chan struct{})
 
-	dcrdClient, err := dcrdWithNotifs.Client(ctx, netParams)
+	dcrdClient, err := dcrdWithNotifs.Client(context.Background(), netParams)
```

Comment:
pass shutdownCtx

Reply:
Using the shutdown context is acceptable here because this client is only used once to register for notifications. It should never be used after a shutdown signal has been received.

I will update the PR.